### DISCUSSION TITLE
功能请求：在Windows上使用类似Android的接口

### DISCUSSION THREAD

**Discussion by sheng-ri** [Tencent/MMKV#1463] [General]
之前就遇到一个场景，使用compose时，是可以在电脑上进行预览的，这个时候mmkv会无法工作。
另个一个就是在电脑上的jvm平台使用，目前的文档是需要自己去编译，没有直接提供java的接口。

**Comment by lingol** [comment]
While we understand its potential need, porting MMKV for JVM in generic (PC, Linux, Mac, etc) is not on our roadmap. 
We appreciate any contributions if you would like to port it.

**Reply by sheng-ri** [reply]
依托于jni和现有android接口可以很快的完成移植工作，但是缺少发布依赖于本机的java库的相关知识，有没有什么比较好的参考....
